src/components/welcome-screen/welcome-screen.scss

```diff
@@ -1,11 +1,62 @@
 @import '../../theme.scss';
 
 .welcomeScreenWrapper {
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  position: absolute;
+  left: 0;
+  top: 0;
   width: 100%;
   height: 100%;
+
+  .background {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.7) 0%, rgba(0, 0, 0, 0) 100%);
+    transform: rotate(-180deg);
+  }
+  .content {
+    color: map-get($theme-colors, 'white');
+    font-size: 14px;
+    position: absolute;
+    bottom: 0px;
+    padding: 32px;
+
+    .title {
+      font-weight: 700;
+      font-size: 32px;
+    }
+    .desc {
+      font-weight: 400;
+      font-size: 14px;
+      padding-top: 16px;
+    }
+    .download {
+      .icon {
+        height: 24px;
+        width: 24px;
+        margin-right: 4px;
+      }
+      &:hover {
+        background-color: #888888;
```

Comment:
background-color: map-get($theme-colors, 'disabled');

Reply:
It's the same color but it is not a disabled color :) Not sure we should use the same theme here